docs/source/04_contributing/02_local_testing.md

```diff
@@ -0,0 +1,16 @@
+(local-testing)=
+# Local testing
+
+## Unit tests
+
+The plugin has unit tests that can be run with `tox`:
+```console
+$ pip install tox-pip-version
+$ tox -v -e py38
```

Comment:
Maybe it should be stated clearly that py38 env is just an example for Python 3.8 not the only option?

Reply:
fixed